bitcoin/script/miniscript.cpp

```diff
@@ -402,24 +402,15 @@ bool ParseScriptNumber(const std::pair<opcodetype, std::vector<unsigned char>>&
     return false;
 }
 
-int FindNextChar(const char*& iter, const char* const& end, const char m)
+int FindNextChar(Span<const char>& sp, const char m)
```

Comment:
Nit: pass Span by value (general advice is that objects no bigger than 2 pointers are more efficient to pass by value than by const reference).